### PR TITLE
Improve COManage instructions post user trial

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

File: docs/policy/comanage-instructions-admin.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+Approving COManage Registrations
+================================
+
+OSG is using a new identity management system called COManage.
+This system will be used for managing access to OSG staff-internal monitoring webpages at UNL
```
Comment (osg-cat) [inline docs/policy/comanage-instructions-admin.md]: Initially, this system …

File: docs/policy/comanage-instructions-admin.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+1.  Check for an email from <registry@cilogon.org> saying "Petition for <NAME> changed status from
+    Confirmed to Pending Approval".
+
+1.  Click the first link in the email.
+
+1.  Log in with your institutional credentials.
+    You should see a "View CO Petition..." page.
+    The Status should be "Pending Approval".
+
+1.  Verify that the application is legitimate.
```
Comment (osg-cat) [inline docs/policy/comanage-instructions-admin.md]: Do you want to provide any guidance here?

File: docs/policy/comanage-instructions-user.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+Registering for the OSG COManage
+================================
+
+OSG is using a new identity management system called COManage.
+This system will be used for managing access to OSG staff-internal monitoring webpages at UNL
```
Comment (osg-cat) [inline docs/policy/comanage-instructions-user.md]: Initially, this system …

File: docs/policy/comanage-instructions-user.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+
+1.  Go to <https://registry.cilogon.org/registry/co_petitions/start/coef:64>
+
+1.  Search for your institution and select it.
+
+1.  Log in with your institutional credentials.
+    You should reach the "OSG Registration: Self Signup With Approval" page.
+
+1.  Click "BEGIN".
```
Comment (osg-cat) [inline docs/policy/comanage-instructions-user.md]: Personally, I prefer a style where you both label and name the UI control, e.g.: Click the “BEGIN” button. But do whatever is consistent with the rest of this documentation area.

File: docs/policy/comanage-instructions-user.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+1.  Enter your name and email address in the form fields;
+    defaults might be filled in from what your institution provided,
+    but you are permitted to change them.
```
Comment (osg-cat) [inline docs/policy/comanage-instructions-user.md]: Add a note about it being OK to use a different email address here than used to login?

File: docs/policy/comanage-instructions-user.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+1.  To verify your access, go to
+    <https://registry.cilogon.org/registry/users/login>.
+
+    If you see a page saying "Welcome to OSG. Please select an action from the menus.",
+    then your registration was successful.
+
+    Otherwise, if you see the "Self Signup With Approval" page, then your registration was unsuccessful;
+    contact <help@opensciencegrid.org> for assistance.
+
+1.  The menus are on the left.
```
Comment (osg-cat) [inline docs/policy/comanage-instructions-user.md]: Note that this step is optional or just to explore the interface.

File: docs/policy/comanage-instructions-admin.md (hunk 2 above)
Comment (matyasselmeci) [reply docs/policy/comanage-instructions-admin.md]: I'd love to but I don't have any guidance myself. Anybody else have ideas?

File: docs/policy/comanage-instructions-user.md (hunk 6 above)
Comment (matyasselmeci) [reply docs/policy/comanage-instructions-user.md]: Should I just drop that step?

File: docs/policy/comanage-instructions-admin.md (hunk 2 above)
Comment (osg-cat) [reply docs/policy/comanage-instructions-admin.md]: Talk to the service owners or the Security team.

File: docs/policy/comanage-instructions-user.md (hunk 6 above)
Comment (osg-cat) [reply docs/policy/comanage-instructions-user.md]: Fine by me.

File: docs/policy/comanage-instructions-admin.md (hunk 2 above)
Comment (matyasselmeci) [reply docs/policy/comanage-instructions-admin.md]: OK, I added some brief instructions.

Comment (brianhlin) [reply docs/policy/comanage-instructions-admin.md]: LGTM

**Pull request merged**